src/app/Assess-Batch/Services/category.service.ts

```diff
@@ -27,10 +27,14 @@ export class CategoryService {
   url = environment.serverRootURL + '/category';
 
   getCategories(): Observable<[Category]> {
-    return this.http.get<[Category]>(`${this.url}/all`, httpOptions);
+    return this.http.get<[Category]>(`${this.url}`, httpOptions);
   }
 
   getCategoryById(id: number): Observable<Category> {
     return this.http.get<Category>(`${this.url}/${id}`, httpOptions);
   }
+
+  getActiveCatgories(): Observable<Category[]> {
+    return this.http.get<Category[]>(environment.api.category.active, httpOptions);
```

Comment:
Question around this. If the plan is to fully remove the URLs from the codebase then line 34 should also have something similar to this. Is that the direction we are going?

Reply:
Or is this just a change until the high level services are done.

Reply:
Correct, I am normalizing the URL's in the high level services - making the environment file as new methods are added to each respected service